Form/LegacyModifySettingsForm.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Craue\ConfigBundle\Form;
+
+/**
+ * for Symfony < 2.8
+ *
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+class LegacyModifySettingsForm extends AbstractModifySettingsForm {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getName() {
```

Comment:
well, it should even work with 2.1 to 2.8 actually, as there is no change between 2.1 and 2.3 on this point